### ISSUE TITLE
'generation': 'INVALID' in evaluated_individuals_

### ISSUE THREAD

**Issue by Genises** [EpistasisLab/tpot#1040]
Hi there, 
looking at tpot.evaluated_individuals_ after calling the fit() function, every individuals generation reads: 'generation': 'INVALID' except for the first generation. That is true for multiple data sets I tested.

My config is always:
TPOTRegressor(generations=100, population_size=100, verbosity=2,
                             config_dict=own_config, scoring="neg_mean_squared_error", max_time_mins=600,
                             max_eval_time_mins=30)

Everything else seems fine;  
'DecisionTreeRegressor(SelectPercentile(FastICA(input_matrix, FastICA__tol=0.05), SelectPercentile__percentile=90))': {'generation': 'INVALID', 'mutation_count': 1, 'crossover_count': 0, 'predecessor': ('DecisionTreeRegressor(FastICA(input_matrix, FastICA__tol=0.05))',), 'operator_count': 3, 'internal_cv_score': -121955.76376014757}

Is this a bug or am I reading something wrong? 
Thank you.

**Comment by weixuanfu** [comment]
Thank you for reporting this bug. This info was set to "INVALID" when the pipeline was generated ([this line](https://github.com/EpistasisLab/tpot/blob/master/tpot/base.py#L1554) and [this one](https://github.com/EpistasisLab/tpot/blob/master/tpot/base.py#L1500)). But it should be reset to the generation that the pipeline was evaluated based on [those lines](https://github.com/EpistasisLab/tpot/blob/master/tpot/gp_deap.py#L245). The bug is that those lines should update pipelines in `offsprings` instead of `population`. I will make a PR and make a fix for dev branch soon.

**Comment by weixuanfu** [comment]
I just made a PR and merged it into development branch of TPOT to add this kernel support. You may try the dev branch via the command below:
```
pip install --upgrade --no-deps --force-reinstall git+https://github.com/EpistasisLab/tpot.git@development
```

**Comment by weixuanfu** [comment]
This issue should be fixed in TPOT v0.11.2. Please feel free to reopen this issue if there is a unsolved issue related to this one.
